### ISSUE TITLE
Buchung ohne Registrierung als Member

### ISSUE THREAD

**Issue by MichaGi** [markocupic/resource-reservation-bundle#33]
Ist eine Buchungsmöglichkeit per einfachem Formular (ohne vorherige Registrierung als Mitglied), wie in der Erweiterung "calendar-event-booking-bundle", realisierbar?

Dies wäre absolut wünschenswert, da man damit auch die einfache Buchung von Zeitslots über längere Zeiträume ohne das lästige Anlegen und Verwalten hunderter Events realisieren könnte.

Vielen Dank im Voraus!

**Comment by markocupic** [comment]
Nein, das von dir angesprochene Feature ist nicht implementiert und wird es ohne konkreten Auftrag auch nicht werden.
